Quantum-Control-Applications/Superconducting/Single-Flux-Tunable-Transmon/17_cryoscope_1ns.py

```diff
@@ -173,7 +173,7 @@ def baked_waveform(waveform, pulse_duration):
                 # Wait some time to ensure that the flux pulse will arrive after the x90 pulse
                 wait(20 * u.ns)
                 with switch_(segment):
-                    for j in range(0, len(flux_waveform) + 1):
+                    for j in range(1, len(flux_waveform) + 1):
```

Comment:
`for j in range(0, len(flux_waveform)):`

Reply:
this has to be `for j in range(0, len(flux_waveform))` because we have modified to be from (1, pulse_len +1) in baking